.github/workflows/test.yml

```
@@ -107,44 +135,172 @@ jobs:
+    strategy:
+      fail-fast: false
+      matrix:
+        consumer: ${{ fromJson(needs.Contract-Tests.outputs.consumers) }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Discover latest result for ${{ matrix.consumer.name }}
+        id: discover
+        shell: bash
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2012:info:7:10: Use find instead of ls to better handle non-alphanumeric filenames

(shellcheck)